### PR TITLE
Changed NPM tag 10-slim to 10.

### PR DESCRIPTION
**Pull request by sweetpalma**
That would help running tools like `lerna`:
https://github.com/actions/npm/issues/8

### CODE REVIEW

**Comment by parkr** [comment]
It looks like the aim of this is to have Git installed. I wonder if it would be a better approach to simply install git in the slim image instead.

**Comment by hfaulds** [comment]
@parkr I think that's reasonable although I wonder if there may be other dependencies required for other use-cases (https://github.com/actions/npm/issues/8#issuecomment-447676538 mentions python for `node-gyp` for example).

I opened https://github.com/actions/npm/pull/20 with another option which is to add a slim version of this action so people can carry on using it if they so wish.

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:10-slim
+FROM node:10
 
 LABEL version="1.0.0"
```
Comment (hfaulds) [inline Dockerfile]: I think we should probably bump this to `1.1.0`, it is a fairly significant change

**Comment by cleishm** [comment]
Looking forward to this being merged. Cheers!

**Pull request merged**